pegasus/sites.v3/code.org/public/educate/professional-learning.md

```diff
@@ -6,39 +6,58 @@ theme: responsive
 # Professional Learning #
 <img src="/images/k5pdcropped.jpg" width="100%"/>
 
+## New to computer science? No worries.
```

Comment:
Yeah since the first part was a question, seemed weird to leave it off. We do this sometimes.